webapp/channels/src/components/widgets/menu/menu_items/menu_start_trial.tsx

```diff
@@ -45,54 +49,42 @@ const MenuStartTrial = (props: Props): JSX.Element | null => {
             dialogType: LearnMoreTrialModal,
         }));
     };
-
-    const openTrialBenefitsModal = () => {
-        trackEvent(
-            TELEMETRY_CATEGORIES.SELF_HOSTED_START_TRIAL_MODAL,
-            'open_trial_benefits_modal_from_menu',
-        );
-        dispatch(openModal({
-            modalId: ModalIdentifiers.TRIAL_BENEFITS_MODAL,
-            dialogType: TrialBenefitsModal,
-        }));
-    };
-
     const prevTrialLicense = useSelector((state: GlobalState) => state.entities.admin.prevTrialLicense);
     const license = useSelector(getLicense);
     const isPrevLicensed = prevTrialLicense?.IsLicensed;
     const isCurrentLicensed = license?.IsLicensed;
-    const isCurrentLicenseTrial = isTrialLicense(license);
 
-    // Show this CTA if the instance is currently not licensed and has never had a trial license loaded before
-    const show = (isCurrentLicensed === 'false' && isPrevLicensed === 'false') || isCurrentLicenseTrial;
-    if (!show) {
+    if (isCurrentLicensed === 'true') {
         return null;
     }
 
+    const showTrialButton = isCurrentLicensed === 'false' && isPrevLicensed === 'false' && hasPermissionForStartTrial && isE0;
+
     return (
         <li
             className={'MenuStartTrial'}
             role='menuitem'
             id={props.id}
         >
-            {isCurrentLicenseTrial ? <>
-                <div style={{display: 'inline'}}>
-                    <span>
-                        {formatMessage({id: 'navbar_dropdown.reviewTrialBenefits', defaultMessage: 'Review the features you get with Enterprise. '})}
-                    </span>
-                    <button onClick={openTrialBenefitsModal}>
-                        {formatMessage({id: 'navbar_dropdown.learnMoreTrialBenefits', defaultMessage: 'Learn More'})}
-                    </button>
-                </div>
-            </> : <>
+            <div className='free_version_badge'>{'FREE EDITION'}</div>
+            {isE0 &&
+                <div className='start_trial_content'>
+                    {formatMessage({
+                        id: 'navbar_dropdown.versionTextE0',
+                        defaultMessage: 'This is the free edition of Mattermost, ideal for evaluation and small teams.',
+                    })}
+                </div>}
+            {!isE0 &&
                 <div className='start_trial_content'>
-                    {formatMessage({id: 'navbar_dropdown.tryTrialNow', defaultMessage: 'Try Enterprise for free now!'})}
-                </div>
+                    {formatMessage({
+                        id: 'navbar_dropdown.versionTextTeamEdition',
+                        defaultMessage: 'This is the free open source edition of Mattermost, ideal for evaluation and small teams.',
```

Comment:
Is this string supposed to mention "team" edition instead of "free"? Or omit "free"?

Reply:
Probably the key there is a bit misleading, I can change it. The text is intentionally speaking about the "free" trait of the Mattermost edition used here.